ipaserver/install/certs.py

```diff
@@ -142,10 +138,8 @@ def passwd_fname(self):
     def __del__(self):
         if self.reqdir is not None:
             shutil.rmtree(self.reqdir, ignore_errors=True)
-        try:
-            os.chdir(self.cwd)
-        except OSError:
-            pass
+            self.reqdir = None
+        self.nssdb.close()
```

Comment:
Personally I find it cleaner to always call close. You never know of NSSDatabase is going to need close() for other reasons.

But I'm not going to fight you on this.